README.md

```diff
@@ -76,23 +76,24 @@ $ cd simple-ctv-vault
 $ pip install -r requirements.txt
 
 # build this bitcoin branch
-#  https://github.com/JeremyRubin/bitcoin/tree/checktemplateverify-rebase-4-15-21
-$ bitcoind -regtest -txindex=1 &
+#  https://github.com/JeremyRubin/bitcoin/tree/checktemplateverify-signet-23.0-alpha
+$ bitcoind -signet -signetchallenge=512102946e8ba8eca597194e7ed90377d9bbebc5d17a9609ab3e35e706612ee882759351ae -signetseednode=50.18.75.225 -txindex=1 &
```

Comment:
General feedback on this PR: we need to both support signet and regtest, so it would be useful to segment the various parts of this change out and keep regtest-specific content intact.